deploy/crds/examples/rhmi.cr.yaml

```diff
@@ -10,3 +10,4 @@ spec:
   smtpSecret: redhat-rhmi-smtp
   deadMansSnitchSecret: redhat-rhmi-deadmanssnitch
   pagerDutySecret: redhat-rhmi-pagerduty
+  priorityClassName: rhoam-pod-priority
```

Comment:
In theory this should be fine, I just wonder should we also change https://github.com/integr8ly/integreatly-operator/blob/master/pkg/controller/installation/installation_controller.go#L60 as part of this to keep the name consistent?

Reply:
no objections from me with that change